node/index.js

```diff
@@ -638,3 +663,138 @@ app.get('/api/signal_evaluate', function (request, response, next) {
     })
     .catch(next);
 });
+
+// Retrieve CRA Base Report and PDF
+// Base report: https://plaid.com/docs/check/api/#cracheck_reportbase_reportget
+// PDF: https://plaid.com/docs/check/api/#cracheck_reportpdfget
+app.get('/api/cra/check_report', function (request, response, next) {
```

Comment:
Some of the endpoint and function names here are a little inconsistent. It's not a big deal, but I'd like to see us align on a pattern here. As a reminder, the category of products here is "Check Report" or "Consumer Report" (these names are admittedly confusing), and the products themselves are Base Report, Income Insights, and Partner Insights. So, I would recommend either prefixing endpoint names with Check Report if that makes organization easier, or just dropping the reference to Check Report altogether and using Income Insights, Base Report, etc.

One place where you can't do this is the PDF endpoint, b/c that endpoint truly is just a getter for Check Report products (Base Report + add-ons). So, maybe this one can stay the same.

Don't spend too much time renaming stuff, but I'd like to see more consistency if possible.